README.md

```diff
@@ -48,7 +48,7 @@ The major components of the Boson Project are:
 
 If you don't already have an OpenShift cluster with Serverless installed, or a
 Kubernetes cluster with Knative Serving and Eventing, you can follow our
-[Getting Started with Kubernetes Guide](https://github.com/boson-project/faas/blob/develop/docs/getting_started_kubernetes.md)
+[Getting Started with Kubernetes Guide](https://github.com/boson-project/func/blob/develop/docs/getting_started_kubernetes.md)
```

Comment:
The link seems not to work, maybe the `develop` branch should be changed to `main`.